src/configuration.h

```diff
@@ -139,8 +139,9 @@ class MetadataAgentConfiguration {
   }
 
  private:
+  friend class MetadataAgentConfigurationTest;
   void ParseConfigFile(const std::string& filename);
-
```

Comment:
Let's keep this blank line to separate functions from instance variables.

Reply:
Replaced.